capture/app.py

```diff
@@ -46,3 +68,24 @@ def main():
         logger.debug(e)
         parser.print_help()
         # raise
+
+    try:
```

Comment:
remember the parse calls the `assemble` function line 66. That's where you should call your split function (and do everything basically.